ports/glutin/window.rs

```diff
@@ -605,15 +607,21 @@ impl WindowMethods for Window {
     }
 
     fn set_page_title(&self, title: Option<String>) {
+        let fallback_title = match self.current_url.borrow_mut().clone() {
```

Comment:
This could be
```rust
let fallback_title = if let Some (ref current_url) = self.current_url.borrow() {
   current_url.clone()
} else {
    String::from("Untitled")
};
```